NOTICE

```diff
@@ -0,0 +1,5 @@
+Jenkins Remoting
+Copyright (c) 2004-2016, Sun Microsystems, Inc., Kohsuke Kawaguchi, Daniel Dyer, Stephen Connolly, and others
+
+This product includes software developed at
```

Comment:
No because the root NOTICE is for remoting.jar whereas the NOTICE in the agent-load-test is for all the files in that, which includes test scoped dependencies